### PR TITLE
Feature s3 bucket

### PR DESCRIPTION
**Pull request by DawKozlowski**

### CODE REVIEW

File: src/main/java/com/construction_worker_forum_back/service/UserService.java [hunk 1]
```diff
@@ -84,20 +92,52 @@ public UserDto changeBio(String username, BioSimpleDto newBio) {
...
+                    .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
+            fileName = user.getId().toString();
+            file = FileUploadUtil.convertMultiPartFileToFile(multipartFile);
+            s3Client.putObject(new PutObjectRequest(bucketName, fileName, file));
+            user.setAvatar(fileName);
+            userRepository.save(user);
+            file.delete();
+        } catch (Exception e) {
+            file.delete();
+            log.trace("Jeblo to jeblo na chuj drazyc temat");
```
Comment (Robert-Michalski) [inline src/main/java/com/construction_worker_forum_back/service/UserService.java]: Plis

Comment (DawKozlowski) [reply src/main/java/com/construction_worker_forum_back/service/UserService.java]: xd

File: src/main/java/com/construction_worker_forum_back/service/UserService.java [hunk 2]
```diff
@@ -17,25 +22,28 @@
...
+import java.io.File;
 import java.io.IOException;
 import java.time.Instant;
-import java.util.Date;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 
 @Service
-@AllArgsConstructor
```
Comment (Robert-Michalski) [inline src/main/java/com/construction_worker_forum_back/service/UserService.java:33]: Why did you get rid of AllArgsConstructor ?

Comment (DawKozlowski) [reply src/main/java/com/construction_worker_forum_back/service/UserService.java:33]: We have value annotation on field which makes it useless. Its better to use RequiredArgsConstructor annotation

**Review by Robert-Michalski — APPROVED** [review]

**Pull request merged**